### PR TITLE
Add and Update ACTIONX Tests

### PR DESCRIPTION
**Pull request by OPMUSER**
Update ACTIONX_COMPLUMP.DATA

### CODE REVIEW

**Comment by OPMUSER** [comment]
All the test decks are complete, only the documentation (ACTIONX.md and ACTIONX.odp) will change as a few bugs are hopefully fixed and a comparison is made with the commercial simulator.

The tests are therefore the tests are ready for review.

**Comment by alfbr** [comment]
Thank you! I will just let jenkins confirm that this does not break any existing tests before merging.

**Comment by alfbr** [comment]
jenkins build this please

**Comment by OPMUSER** [comment]
ACTIONX_WCONINJE was missing the restart files in the previous submission:

https://github.com/OPM/opm-tests/pull/700/commits/ec64fb30661dc7c33aa600afe7f350cd3eba33c4

File: actionx/ACTIONX_WCONINJE.DATA [hunk 1]
```diff
@@ -322,6 +322,8 @@ RPTSCHED
 --
 --       RESTART CONTROL BASIC = 4 (ALL=2, YEARLY=4, MONTHLY=5, TSTEP=6)
 --
+RPTRST
+         BASIC=5                                                               /
```
Comment (bska) [inline actionx/ACTIONX_WCONINJE.DATA:326]: Note that this particular change reverts PR #635.  Is that intentional?

Comment (OPMUSER) [reply actionx/ACTIONX_WCONINJE.DATA:326]: Yes.

Comment (bska) [reply actionx/ACTIONX_WCONINJE.DATA:326]: >> Note that this particular change reverts PR #635. Is that intentional?
>
> Yes.

In that case I'll have @joakim-hove do the review here.  I don't think the underlying issue that prompted #635 has been fixed in the interim.

Comment (alfbr) [reply actionx/ACTIONX_WCONINJE.DATA:326]: > > > Note that this particular change reverts PR #635. Is that intentional?
> > 
> > 
> > Yes.
> 
> In that case I'll have @joakim-hove do the review here. I don't think the underlying issue that prompted #635 has been fixed in the interim.

Yes, and the change also breaks the test question. Highly appreciated if @joakim-hove takes over the review.

Comment (joakim-hove) [reply actionx/ACTIONX_WCONINJE.DATA:326]: Well; for the `ACTIONX_WCONINJE` I can only confirm that the `RPTRST BASIC=5` issue has not been fixed - getting the `RPTRST / RPTSCHED` functionality work has been *extremely* difficult :disappointed: 

Now I agree that the `BASIC=5` should eventually be fixed - but is there a real value in enforcing that for this particular test? I would encourage you to just revert this particular change.

**Comment by OPMUSER** [comment]
Will do,

File: actionx/ACTIONX_WCONINJE.DATA (hunk 1 above)
Comment (OPMUSER) [reply actionx/ACTIONX_WCONINJE.DATA]: Done

**Comment by blattms** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
Thanks.  I'll merge into master.

**Pull request merged**